mvcc/kvstore_txn.go

```diff
@@ -15,10 +15,12 @@
 package mvcc
 
 import (
-	"go.etcd.io/etcd/v3/lease"
-	"go.etcd.io/etcd/v3/mvcc/backend"
-	"go.etcd.io/etcd/v3/mvcc/mvccpb"
-	"go.etcd.io/etcd/v3/pkg/traceutil"
+	"sort"
+
+	"go.etcd.io/etcd/lease"
+	"go.etcd.io/etcd/mvcc/backend"
+	"go.etcd.io/etcd/mvcc/mvccpb"
```

Comment:
Use `go.etcd.io/etcd/v3` replace `go.etcd.io/etcd` please. That is the rootcause of CI failed like https://travis-ci.com/github/etcd-io/etcd/jobs/385763253 .

Reply:
yeah... thank u very much